examples/multimap/main.go

```diff
@@ -52,5 +52,11 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	fmt.Printf("entry found:%t", ok)
+	fmt.Printf("entry found:%t\n", ok)
+	// Print number of values match given "key"
```

Comment:
I think it should be `// Print number of values matching given "key"` or `// Print number of values that match the given "key"`

Reply:
[4df6cf3](https://github.com/hazelcast/hazelcast-go-client/pull/734/commits/4df6cf34201606cf13235de430fa990af177b711).